tsconfig.json

```diff
@@ -5,9 +5,10 @@
 		"@tsconfig/strictest/tsconfig.json"
 	],
 	"compilerOptions": {
+		"moduleResolution": "node16",
+		"module": "Node16",
```

Comment:
Aren't those already properly set by `node-lts`?

Reply:
No typescript errored out for no reason?

Reply:
Huh, that's weird... These should be set by the node-lts preset https://github.com/tsconfig/bases/blob/main/bases/node-lts.json

Reply:
I will revert the ts config and you can try figuring it out. Everytime I mess with the ts config I want to bang my head against a wall.

Reply:
I feel you! I've stopped worrying about this an use tsbases for this very reason 😄

If you need a helping hand I can help you troubleshoot, feel free to comment here with the details of the problem. 🙂

Reply:
After testing isolatedModules can be false and everything works.  
  
Then by putting the `"@tsconfig/node-lts/tsconfig.json"` on the list of extends it fixes the   
module Resolution Error